### ISSUE TITLE
Interface names are static

### ISSUE THREAD

**Issue by Akhil-Jalagam** [esc0rtd3w/wifi-hacker#12]
Interface name are not always wlan0,1,..etc

wlo1 is also an wireless interface

**Comment by esc0rtd3w** [comment]
yeah, that was mainly for testing after the switch from Kali 1.xx to Kali 2.xx.

i can make them dynamic in the 2.0 release, i will have time soon to work on it. Been busy with other projects lately.

**Comment by BitCoder444** [comment]
How do you fix this? Maybe you could get the output from ifconfig and then parse that to get all interface names

**Comment by esc0rtd3w** [comment]
Have not looked into it much. May do that soon and try your suggestion

**Comment by esc0rtd3w** [comment]
Can change name (wlan0, wlan0mon, etc) with variable in script.

**Comment by MoBattah** [comment]
If I have time next week, I'll fork it, make it a variable (interactively ask) then PR.

**Comment by Syslog777** [comment]
Cool
